app/templates/start.html

```diff
@@ -77,6 +77,7 @@ <h1 class="ons-u-fs-xxl ons-u-mt-l">{{_('Start study')}}</h1>
     }}
 
     {{ onsButton({
+        'id': 'submitUACBtn',
```

Comment:
We're using underscore separated word IDs elsewhere, good to stick to convention 💅